### PR TITLE
Added .pyi file for flatten

### PR DESCRIPTION
**Pull request by Chillee**
Generated with `stubgen` and moved from `out/flatten.pyi` to `flatten.pyi.in`.

https://github.com/pytorch/pytorch/pull/22245#issuecomment-521875658

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by janosh** [comment]
I'm having a similar problem as @PistonY in https://github.com/pytorch/pytorch/pull/22245#issuecomment-521875658. Pylance is unable to find `nn.Flatten`.

Softmax working

Softmax working

Flatten not working

Flatten not working

**Comment by ezyang** [comment]
@janosh please file a new issue on this; also, please check if the issue still occurs on nightly.

**Comment by janosh** [comment]
@ezyang Sorry for the noise. I was on 1.5. Updating to 1.6 fixed the problem.
